### DISCUSSION TITLE
Are repository names with slash '/' character allowed?

### DISCUSSION THREAD

**Discussion by domfe** [quay/quay#3069] [General]
Hello.
We are using repository names with '/' inside ("example/mydebian", excluding organization).
The name is working well with pull, push and everything inside quay except mail notification.
The log shows an error in util/jinjautil.py on line 51 because split("/") function is expecting 2 parts, organization (namespace) and repository, but our repo name is, in fact, producing 3 parts ("myorganization/example/mydebian").
So, are slash '/' chars allowed in repository names (quay doesn't complain when creating such repos)?
I think the error with email notification could be prevented using split("/", 2) in line 51 and line 69.

Thanks

**Comment by bcaton85** [comment]
Good catch. Would you be willing to open a PR for this? We would gladly approve it. Currently our developers are busy on higher priority tasks so it will take some time to get picked up.
